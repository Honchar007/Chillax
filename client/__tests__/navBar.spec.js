import React from 'react'
import renderer from 'react-test-renderer'
import NavBar from '../components/Navbar/Navbar'

it('renders correctly', () => {
  const tree = renderer.create(<NavBar></NavBar>).toJSON()
  expect(tree).toMatchSnapshot()
})
