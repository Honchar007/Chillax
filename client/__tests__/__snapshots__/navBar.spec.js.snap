// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="NavbarElements__Nav-a01551-0 clQdYN"
>
  <a
    className="NavbarElements__NavLink-a01551-1 kbLYqq"
    href="/home"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <img
      alt="logo"
      className="logo"
      src="https://api.freelogodesign.org/files/34eb4009482b4c919fdfaf4f64249cd9/thumb/logo_200x200.png?v=637534660950000000"
    />
  </a>
  <svg
    className="NavbarElements__Bars-a01551-2 eOQQfC"
    fill="currentColor"
    height="1em"
    stroke="currentColor"
    strokeWidth="0"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 448 512"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
    />
  </svg>
  <div
    className="NavbarElements__NavMenu-a01551-3 jRqXJs"
  >
    <a
      className="NavbarElements__NavLink-a01551-1 kbLYqq"
      href="/exploration"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Explore
    </a>
    <a
      className="NavbarElements__NavLink-a01551-1 kbLYqq"
      href="/messanger"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Messanger
    </a>
    <a
      className="NavbarElements__NavLink-a01551-1 kbLYqq"
      href="/parties"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      Patries
    </a>
  </div>
  <nav
    className="NavbarElements__NavBtn-a01551-4 jREmHR"
    onClick={[Function]}
  >
    <a
      className="NavbarElements__NavBtnLink-a01551-5 ehwJar"
    >
      Sign in
    </a>
  </nav>
</nav>
`;
